Stratis.Bitcoin/Features/Wallet/IWalletManager.cs

```diff
@@ -37,11 +37,10 @@ public interface IWalletManager : IDisposable
 
         /// <summary>
         /// Loads a wallet from a file.
-        /// </summary>
-        /// <param name="password">The user's password.</param>
+        /// </summary>        
         /// <param name="name">The name of the wallet.</param>
         /// <returns>The wallet.</returns>
-        Wallet LoadWallet(string password, string name);
+        Wallet LoadWallet(string name);
```

Comment:
At the moment, the password is not used, but that's because we're not encrypting the wallets yet.
That's because we want to have quick access to the file so that it's easier to debug.
So the password was there as a placeholder really, because we know we'll need it in the future.